diff --git a/node_modules/react-native-render-html/src/HTMLRenderers.js b/node_modules/react-native-render-html/src/HTMLRenderers.js
index 8c3e394..87eadfd 100644
--- a/node_modules/react-native-render-html/src/HTMLRenderers.js
+++ b/node_modules/react-native-render-html/src/HTMLRenderers.js
@@ -1,5 +1,6 @@
 import React from 'react';
-import { TouchableOpacity, Text, View, WebView, Platform } from 'react-native';
+import { TouchableOpacity, Text, View, Platform } from 'react-native';
+import { WebView } from 'react-native-webview';
 import { _constructStyles, _getElementClassStyles } from './HTMLStyles';
 import HTMLImage from './HTMLImage';
 